packages/relay/src/durable-object.ts

```
@@ -0,0 +1,173 @@
+
+    // Close existing connection (1 connection limit)
+    if (this.ws) {
+      try {
+        this.ws.close(1000, "replaced by new connection");
+      } catch {
+        // already closed
+      }
+      this.ws = null;
+    }
+
+    const pair = new WebSocketPair();
```

ESLint

[error] 'WebSocketPair' is not defined.

(no-undef)